emergence_lib/src/cursor.rs

```diff
@@ -54,7 +54,7 @@ impl Default for CursorWorldPos {
 
 /// The tile position of the mouse cursor, if it lies over the map.
 #[derive(Resource, Default, Clone, Copy, Deref, DerefMut)]
-pub struct CursorTilePos(Option<TilePos>);
+pub struct CursorTilePos(pub Option<TilePos>);
```

Comment:
This probably wants a reader method, rather than being pub. Having this messed with externally will be very confusing.

Reply:
Done, I replaced the Deref stuff with reader methods.